### PR TITLE
fix: Set default value of portal.allServerNames to ${portal.server}

### LINKED ISSUE #1372: Login failure with uPortal 5.3.0 + CAS
**Describe the bug**

Upgrading to uPortal 5.3.0, I'm unable to successfully log in using CAS authentication

Using the bundled CAS with uPortal-start, I increased the logging level and saw the following
```xml
<cas:serviceResponse xmlns:cas='http://www.yale.edu/tp/cas'>
        <cas:authenticationFailure code='INVALID_SERVICE'>
                ticket &#039;ST-1-JTrOtMpLdXOFzIcfnp7J-cas01.example.org&#039; does not match supplied service.  The original service was &#039;http://localhost:8080/uPortal/Login&#039; and the supplied service
was &#039;http://localhost:8080 /uPortal/Login&#039;.
        </cas:authenticationFailure>
</cas:serviceResponse>
```
It seems that there is a whitespace after `http://localhost:8080` but I can't seem to find where the whitepace is coming from

**To Reproduce**
1. Deploy HEAD of uPortal-start
2. Attempt to log in

**Platform:**
 - uPortal Version: uPortal 5.3.0

### PR DESCRIPTION
**Pull request by jonathanmtran**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

Set the default value of `portal.allServerNames` to `${portal.server}`

Resolves #1372 

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: uPortal-webapp/src/main/resources/properties/security.properties [hunk 1]
```diff
@@ -38,14 +38,14 @@ portal.context=/uPortal
 portal.protocol.server.context=${portal.protocol}://_CURRENT_SERVER_NAME_${portal.context}
 portal.login.url=${portal.protocol.server.context}/Login
 
-# Needed for proxy-cas in load-balanced system like apache to give back the proxy-ticket on the server that requested it
-# it requires that the load-balanced instance has his serverName.
+# Needed for proxy-cas in load-balanced system like apache to give back the
+# proxy-ticket on the server that requested it it requires that the
```
Comment (jgribonvald) [inline uPortal-webapp/src/main/resources/properties/security.properties]: Ponctuation is needed: ``` . It requires```

Comment (jgribonvald) [reply uPortal-webapp/src/main/resources/properties/security.properties]: Else sorry that you encountered a problem with my commit, and thanks to fix it !
Now I have to purpose a fix on the CAS client to trim white-spaces !

**Comment by jgribonvald** [comment]
LGTM.

**Comment by jonathanmtran** [comment]
I slightly tweaked the doc. Hopefully it is still correct

File: uPortal-webapp/src/main/resources/properties/security.properties [hunk 2]
```diff
@@ -38,14 +38,14 @@ portal.context=/uPortal
 portal.protocol.server.context=${portal.protocol}://_CURRENT_SERVER_NAME_${portal.context}
 portal.login.url=${portal.protocol.server.context}/Login
 
-# Needed for proxy-cas in load-balanced system like apache to give back the proxy-ticket on the server that requested it
-# it requires that the load-balanced instance has his serverName.
+# Needed for proxy-cas in load-balanced system like Apache to return the
+# proxy-ticket to the requesting server. It requires that the load-balanced
+# instance has this serverName.
```
Comment (jgribonvald) [inline uPortal-webapp/src/main/resources/properties/security.properties]: => ```It requires that the load-balanced instance has his serverName``` warning it's not ***this*** but ***his***

After I can understand my English isn't really good. I meen that in load-balanced system there is a common serverName and each instance has his owned serverName  (ie there is a https://my.univ.edu/ for common serverName and after each tomcat have a different servername like https://portal1.univ.edu/ for the first tomcat, https://portal2.univ.edu/ for the second... etc, it's to be able to come back with the proxy ticket on the server that requested it when we don't have shared cluster).

Comment (jonathanmtran) [reply uPortal-webapp/src/main/resources/properties/security.properties]: How's this? If it looks good I'll update the commit 

```
# Needed for proxy-cas in load-balanced system like Apache to return the
# proxy-ticket to the requesting server. It requires that the load-balanced
# instance has his serverName
# Example: my.univ.edu
portal.lbServerName=${portal.server}

# All server names values for multi server name management, separator is a space
# This property should be set/overridden in PORTAL_HOME/uPortal.properties
# Example: portal1.univ.edu portal2.univ.edu
portal.allServerNames=${portal.server}
```

Comment (jgribonvald) [reply uPortal-webapp/src/main/resources/properties/security.properties]: All is good :+1:

**Pull request merged**